app/lib/admin/metrics/dimension/software_versions_dimension.rb

```diff
@@ -98,7 +98,7 @@ def imagemagick_version
   end
 
   def ffmpeg_version
-    version = `ffmpeg -version`.match(/ffmpeg version ([\d\.]+)/)[1]
+    version = `ffmpeg -version`.match(/ffmpeg version ([\w\d\.-]+) (?=Copyright)/)[1]
```

Comment:
For people wanting to merge this: "Copyright" doesn't need to be a lookahead, so:

```ruby
version = `ffmpeg -version`.match(/ffmpeg version ([\w\d\.-]+) Copyright/)[1]
```